proto2ros/proto2ros/equivalences.py

```diff
@@ -389,11 +390,14 @@ def compute_equivalence_for_message(
             oneof_fields.append(which_field)
 
             oneof_message_spec = MessageSpecification(
-                oneof_type.pkg_name, oneof_type.type, oneof_fields, oneof_constants
+                oneof_type.pkg_name,
+                oneof_type.type,
+                oneof_fields,
+                oneof_constants,
             )
             oneof_message_spec.annotations["proto-type"] = protofqn(source, location) + f"[one-of {oneof_decl.name}]"
             oneof_message_spec.annotations["proto-class"] = "one-of"
-            oneof_message_spec.annotations["tagged"] = list(zip(oneof_constants[1:], oneof_fields[:-1]))
+            oneof_message_spec.annotations["tagged"] = list(zip(oneof_constants[1:], oneof_fields[:-1], strict=True))
```

Comment:
@amessing-bdai it fails the `strict` check because this should have been:
```suggestion
            oneof_message_spec.annotations["tagged"] = list(zip(oneof_constants[1:], oneof_fields[:-2], strict=True))
```
Code isn't broken precisely because `zip` stops when `oneof_constants` are exhausted.

Reply:
Sure. In this case, the code still worked, but there are plenty of cases where it causes bugs, so I'm a fan of that ruff rule. I've made the edit.